### PR TITLE
Add installation doc for mattermost on cloudron

### LINKED ISSUE #2284: Add Cloudron as deployment option
Hi Mattermost team,
We have been maintaining the mattermost package on Cloudron for over 2 years now (all the way for mattermost 1.4!) at https://git.cloudron.io/cloudron/mattermost-app. Would be great if you could add us as one of the deployment options.

You are welcome try it out in our demo at https://my-demo.cloudron.me (username: cloudron password: cloudron).

### PR DESCRIPTION
**Pull request by gramakri**
Fixes #2284

### CODE REVIEW

**Comment by amyblais** [comment]
Added a note that this is a community maintained doc.

File: source/install/deploy-cloudron.rst [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+.. image:: https://cloudron.io/img/button.svg
+   :target: https://cloudron.io/button.html?app=org.mattermost.cloudronapp
+
+Installation
+============
+
+To get started, first `install Cloudron <https://cloudron.io/get.html>`_ on a VPS provider like Digital Ocean
+and then install Mattermost from the `Cloudron Store <https://cloudron.io/store/org.mattermost.cloudronapp.html>`_.
+
+The Mattermost app will be already setup to use MySQL. Email sending should also work out of the box.
```
Comment (jasonblais) [inline source/install/deploy-cloudron.rst]: We could simplify these two sentences to
```
The Mattermost app will be configured with MySQL and SMTP email notifications service.
```

**Comment by gramakri** [comment]
Hey, thanks I will update this PR with the requested changes shortly

**Comment by gramakri** [comment]
I have updated it.

**Comment by jasonblais** [comment]
Thank you @gramakri!

**Pull request merged**